torch/distributed/_tensor/debug/visualize_sharding.py

```diff
@@ -154,4 +154,5 @@ def visualize_sharding(dtensor):
     # Convert offsets to blocks with row_ranges for tabulate
     blocks = _convert_offset_to_ranges(all_offsets)
     if device_mesh.get_rank() == 0:
+        print(prompt)
```

Comment:
I think in addition to printing the header, I wonder if we should print some metadata first before printing the actual visualization, i.e. sth like:
dtensor shape: (...), device mesh: (...), placements: (...)

Reply:
would that be optional to user? 
```
def visualize_sharding(..., print_metadata=False):
    ...
    if print_metadata:
        print(shape)
        print(mesh)
        ...
```